File: DomZaStaraLicaApi/Endpoints/TerapijaLijek/Update/TerapijaLijekUpdateRequest.cs
namespace DomZaStaraLicaApi.Endpoints.TerapijaLijek.Update
{
    public class TerapijaLijekUpdateRequest
    {
        public int TerapijaId { get; set; }
        public string Opis { get; set; }
        public int DoktorId { get; set; }
        public int KorisnikDomaID { get; set; }
        public string NacinPrimjene { get; set; }
        public string VremenskiInterval { get; set; }

        public List<int> Lijekovi { get; set; }
    }
}

File: frontend/src/app/terapija/terapijaLijekUpdateRequest.ts
export  interface TerapijaLijekUpdateRequest
{
  terapijaId: number;
  opis: string;
  doktorId: number;
  korisnikDomaID: number;
  nacinPrimjene: string;
  vremenskiInterval: string;
  lijekovi: number[];
}
